spotbugs-exclude.xml

```diff
@@ -0,0 +1,50 @@
+<FindBugsFilter>
```

Comment:
nit: should we add the DTD ? to make it a valid xml file.

```xml 
<?xml version="1.0" encoding="UTF-8"?>
 <FindBugsFilter
              xmlns="https://github.com/spotbugs/filter/3.0.0"
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="https://github.com/spotbugs/filter/3.0.0 https://raw.githubusercontent.com/spotbugs/spotbugs/3.1.0/spotbugs/etc/findbugsfilter.xsd">
```

Reply:
sadly they haven't changed the root of the xml to spotbugsfilter :(

Reply:
https://github.com/cerner/beadledom/commit/3dcd2eabc4d96ed1c9547a1bffc19da4ecf92574